default.yaml

```diff
@@ -23,29 +23,93 @@ exercises:
   minReps: 15
   maxReps: 20
   units: rep
+  info: |
+    *How to set up for a proper push up*
+
+    When it comes to push ups, your form is crucial. Each push up needs to be done perfectly so that your total reps measured from workout to workout are on equal footing. If you did thirty perfect push ups two days ago, and then today you did sixty push ups by only going down halfway, sticking your ass up in the air, etc., it's absolutely impossible to tell if you got any stronger.
+
+    Here's how to get set up to do a push up:
+
+    - When down on the ground, set your hands at a distance that is slightly wider than shoulder-width apart. Depending on your strength and experience, your hands should be angled in a way that feels comfortable to you.  For me, my hands are set up so that my middle finger points straight up and away from me.  You can also turn your hands inwards slightly if it's less stressful on your wrists, or you can do your push ups on your knuckles (as long as you're on a semi-soft surface like grass or carpet.
```

Comment:
nit: Hands should not be rotated. It reduces the ability to generate torque and increases the chance of improper elbow/wrist alignment.